app/serializers/invited_serializer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class InvitedSerializer < ApplicationSerializer
+  attributes :invites, :can_see_invite_details
+
+  def invites
```

Comment:
Can you not use a `has_many` here?

Reply:
I can't find a way to pass the `show_emails` option when using has_many.